Orbitersdk/samples/ProjectApollo/src_csm/saturn.cpp

```diff
@@ -2692,24 +2694,158 @@ void Saturn::GenericTimestep(double simt, double simdt, double mjd)
 	}
 #endif // TRACK_TIMESTEPS
 
-	//
-	// Reduce jostle.
-	//
+	// Visualizing vibration
+	// This simulation of the vibration environment in the CM is mostly based on
+	// Saturn V Launch Vehicle Flight Evaluation Report - AS-506, Apollo 11 Mission
+	double dynpress = GetDynPressure();
+	VECTOR3 vAccel, vWeight, vAngAcc;
+	GetForceVector(vAccel);
+	GetWeightVector(vWeight);
+	
+	GetAngularAcc(vAngAcc);
```

Comment:
vAngAcc seems to not be used by anything.

Reply:
Oops, I forgot to remove the remnants of some experimenting with the angular acceleration. My goal was to replicate the sideways zig-zagging of the rocket during the S-IC burn (see e.g. some comments of Bill Anders or Mike Collins on their Saturn V experience). But it turned  out that in the simulation the F1 gimbaling is way too smooth to cause any significant angular acceleration. Maybe the real-life gimbal was much coarser and more hectic or the more rearward CG caused larger swaying at the top of the rocket, I don't know. Anyway it didn't make any perceptible difference, so I decided to get rid of the code for this. I just simply overlooked the query of angular acceleration, but it is indeed some obsolete code. Can you ignore this line during merge or you want me to make a new commit and PR with vAngAcc stuff is removed?

Reply:
There are three lines where vAngAcc appears (Its declaration, GetAngularAcc and one commented) so it's probably best if you make a small commit to remove it. After that I see no obstacle anymore for getting this PR merged.

Reply:
@ggalfi, just so you know. If you make that change that @indy91 requested, and just commit and push in this vc branch, that's all you should need do. No need to make a new branch or pull request.

Reply:
> No need to make a new branch or pull request.
Good to know, I almost removed this PR, when I've seen your comment.

Reply:
I have done the code cleanup, and while I discovered another minor bug, I adjusted the logic  a little bit. I've tested the new code in the usual non-weightless (weightful ;) ) flight phases.

Reply:
I'll go ahead and merge the PR then. Awesome work!